react/features/toolbox/components/web/HangupMenuButton.tsx

```diff
@@ -88,9 +94,11 @@ class HangupMenuButton extends Component<Props> {
                     onClose = { this._onCloseDialog }
                     placement = 'top-end'>
                     <HangupToggleButton
+                        buttonKey = 'hangup'
```

Comment:
Shouldn't this be "hangup-menu" ? To disciminate between that and the button that does the actual hangup.

Reply:
That's what I've been debating in my head.

Ideally yes since `hangup-menu` conceptually makes more sense, but practically that would mean any app currently relying on the 'hangup' event will no longer work when old Hangup button swaps over to the new one since the buttons in leave menu do not currently trigger any events. 

Adding triggers to those buttons take a bit more work since they are not subclasses of `AbstractButton`. So that means either reimplement those buttons to inherit same behaviour, or we duplicate/extract the [_onClick](https://github.com/jitsi/jitsi-meet/blob/6b260c27a5f42fd0b5a90cd48c8d3299f051930e/react/features/base/toolbox/components/AbstractButton.js#L282) logic to be used in the new buttons.

I did not want to go down that route without first consulting you 😅 

So it came down to:
a) Implement a non-ideal short-term solution that still kinda works for existing apps, and worry about "proper" solution later
b) Go the whole hog and fix all the buttons so we trigger "hangup-menu` on the button in the toolbar, and "hangup" and "end-meeting" for the actual buttons.

This PR does (a) since I wanted to get a "good enough" solution before the next release, and then maybe look at a proper fix next.

Thoughts?

Reply:
P.S. happy to change this PR to do just "hangup-menu" for now. Doesn't help with backward compatibility of existing apps, but easy enough for folks to make changes to start listening to both events for now.

Then perhaps someone with skills than yours truly can do a proper fix for the other buttons.

Reply:
BTW, `toolbarButtons: ["hangup"]` still applies to both the old hangup button and the hangup menu button. Are we OK with the discrepancy between toolbarButtons and buttonsWithNotifyClick?